igor-web/src/main/groovy/com/netflix/spinnaker/igor/build/BuildController.groovy

```diff
@@ -178,8 +179,8 @@ class BuildController {
           if (buildService.metaClass.respondsTo(buildService, 'stopQueuedBuild')) {
             buildService.stopQueuedBuild(queuedBuild)
           }
-        } catch (RetrofitError e) {
-          if (e.response?.status != NOT_FOUND.value()) {
+        } catch (SpinnakerServerException e) {
```

Comment:
Since all this does is catch and re-throw, it's sufficient to catch SpinnakerHttpException here, yes?